docker-compose.yml

```diff
@@ -1,7 +1,14 @@
 version: '3.4'
 
+x-php-container-volumes:
+  &php-container-volumes
+    volumes:
+        - ./:/project:delegated
```

Comment:
Is it a problem that the `cli` is currently running as `root` and `php-fpm` is currently running as `www-data`? However, the cache warmup is also running as `root`. It's probably better if everything runs as `www-data`.